packages/domain-picker/src/domain-picker/suggestion-item.tsx

```diff
@@ -13,21 +12,26 @@ type DomainSuggestion = import('@automattic/data-stores').DomainSuggestions.Doma
 /**
  * Internal dependencies
  */
-import { STORE_KEY } from '../../stores/onboard';
-import { FLOW_ID } from '../../constants';
-import { RecordTrainTracksEventProps } from '../../lib/analytics';
-import { getSignupDomainsSuggestionsVendor } from '../../utils/domain-suggestions';
+// import { getSuggestionsVendor } from 'lib/domains/suggestions';
 
-const DOMAIN_SUGGESTION_VENDOR = getSignupDomainsSuggestionsVendor();
+// const DOMAIN_SUGGESTION_VENDOR = getSuggestionsVendor( true );
+
+// TODO: Fix this.
+const DOMAIN_SUGGESTION_VENDOR = 'variation4_front';
 
 interface Props {
 	suggestion: DomainSuggestion;
 	isRecommended?: boolean;
 	isSelected?: boolean;
 	onSelect: ( domainSuggestion: DomainSuggestion ) => void;
 	railcarId: string | undefined;
-	recordAnalytics?: ( event: RecordTrainTracksEventProps ) => void;
+	// TODO: How to use analytics type here?
+	// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+	// @ts-ignore
```

Comment:
Tagging TODO :-)

Reply:
Using plain ol' object for analytics event props. https://github.com/Automattic/wp-calypso/pull/42409/commits/2e4ae7ee2c74e00449cb75b1ee1da53181136d14